### DISCUSSION TITLE
Zen options to disable line numbers?

### DISCUSSION THREAD

**Discussion by benfrain** [folke/snacks.nvim#265] [Q&A] [answered]
Looking to emulate the setup I had for zen-mode in the Snacks.zen

I want to disable the line numbers. Is this possible currently?

Here is my current config, none of the `show` options (which I had set in zen-mode currently work):

```lua
  zen = {
    minimal = true,
    win = {
      backdrop = {
        transparent = false,
      },
    },
    show = { number = false, signcolumn = "no", cursorcolumn = false, relativenumber = false },
  },
```

**Comment by folke** [comment]
Yes it's possible. Check the docs.
I write documentation for a reason.

`zen-mode` is **NOT** `snacks.zen`. Different functionality and different way to configure.

Replicating a similar config is up to the user. I'm not going to manually convert everyone's config...

**Reply by benfrain** [reply]
I'm looking at https://github.com/folke/snacks.nvim/blob/main/docs/zen.md and there is no mention of how to disable the line numbers?

If it is there and I simply don't understand it, I'm happy to try and add a section in the docs that might help others like me understand it more easily

**Reply by folke** [reply]
You can add **any** snacks toggle here.
![image](https://github.com/user-attachments/assets/29a8e76b-b4e7-4248-ab38-23de6ce6a44d)

You can also configure the `zen` window style with `opts.styles.zen...`. Check the `win` docs for all window options you can change.

**Reply by folke** [reply, marked as answer]
So `opts.styles.zen.wo = {add your window options here}`

**Reply by benfrain** [reply]
Would it be helpful if I added some examples in that Readme or would you rather keep it as is? Appreciate I might be slower than the average user ;)
